oak/server/rust/oak_config_serializer/src/main.rs

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! An utility for serializing an Oak application configuration.
+//!
+//! Invoke with:
+//!
+//! ```shell
+//! cargo run --manifest-path=oak/server/rust/oak_config_serializer/Cargo.toml -- \
+//!     --config-file=<CONFIGURATION_FILE> \
+//!     --output-file=<OUTPUT_FILE>
+//! ```
+
+use anyhow::Context;
+use log::debug;
+use oak_abi::proto::oak::application::{
+    node_configuration::ConfigType, ApplicationConfiguration, NodeConfiguration,
+    WebAssemblyConfiguration,
+};
+use oak_utils::io::{read_file, write_file};
+use prost::Message;
+use std::{collections::HashMap, fs::read_to_string};
+use structopt::StructOpt;
+
+/// Command line options for the Oak Application Configuration Serializer.
+#[derive(StructOpt, Clone, Debug)]
+#[structopt(about = "Oak Application Configuration Serializer")]
+struct Opt {
+    #[structopt(long, help = "TOML application configuration file")]
+    config_file: String,
```

Comment:
Specify somewhere that it is the input file (either in the field name, or in the help)